src/__test__/__snapshots__/categories.test.js.snap

```diff
@@ -0,0 +1,209 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Testing Home Page: <CategoriesList /> component Should match the snapshot 1`] = `
+Array [
```

Comment:
- [ ] great job with the test. Please, like the previous reviewer noted, kindly add unit tests for some pure functions in Redux code. However, all the test I see is unit tests for the components such as booklist and the categorylist. Kindly ensure to add unit tests for some of the pure functions in Redux code. 🙏